.gitignore

```diff
@@ -14,4 +14,113 @@ testResults
+EU4toV2/CMakeFiles/EU4ToVic2.dir/Source/Mappers/IdeaEffectMapper.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/Source/Mappers/MinorityPopMapper.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/Source/Mappers/ProvinceMapper.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/Source/Mappers/ReligionMapper.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/Source/Mappers/SlaveCultureMapper.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/Source/Mappers/StateMapper.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/Source/V2World/BlockedTechSchools.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/Source/V2World/V2Army.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/Source/V2World/V2Country.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/Source/V2World/V2Creditor.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/Source/V2World/V2Diplomacy.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/Source/V2World/V2Factory.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/Source/V2World/V2Flags.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/Source/V2World/V2Leader.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/Source/V2World/V2LeaderTraits.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/Source/V2World/V2Localisation.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/Source/V2World/V2Party.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/Source/V2World/V2Pop.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/Source/V2World/V2Province.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/Source/V2World/V2Reforms.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/Source/V2World/V2Relations.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/Source/V2World/V2State.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/Source/V2World/V2TechSchools.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/Source/V2World/V2World.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/Source/V2World/Vic2CultureUnion.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/Source/V2World/Vic2CultureUnionMapper.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/Source/V2World/Vic2Regions.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/Source/V2World/Vic2TechSchool.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/home/fletcher/EU4toVic2/common_items/CardinalToOrdinal.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/home/fletcher/EU4toVic2/common_items/Color.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/home/fletcher/EU4toVic2/common_items/CommonUtils.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/home/fletcher/EU4toVic2/common_items/Date.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/home/fletcher/EU4toVic2/common_items/LinuxUtils.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/home/fletcher/EU4toVic2/common_items/Log.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/home/fletcher/EU4toVic2/common_items/NewParserToOldParserConverters.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/home/fletcher/EU4toVic2/common_items/Object.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/home/fletcher/EU4toVic2/common_items/ParadoxParser8859_15.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/home/fletcher/EU4toVic2/common_items/ParadoxParserUTF8.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/home/fletcher/EU4toVic2/common_items/ParserHelpers.cpp.o
+EU4toV2/CMakeFiles/EU4ToVic2.dir/home/fletcher/EU4toVic2/common_items/newParser.cpp.o
+EU4toV2/Copy_Files.sh

```

Comment:
Likewise

Reply:
I thought i fixed that already. Ill get ti it tonight.

Reply:
done